config.js

```diff
@@ -4,9 +4,11 @@ const path = require('path');
 
 var config = {};
 
-config.webPort = process.env.WEB_PORT || 8000;
+config.webPort = process.env.WEB_PORT || 8002;
```

Comment:
No need to change the default port to 8002. Users can change their port with the env file.

Reply:
I had a server running on this port on my dev machine already.  I thought I had changed that back before doing the pull request.  Sorry.  I fixed it now.